jobs/payment-jobs/tasks/direct_pay_automated_refund_task.py

```diff
@@ -82,9 +82,6 @@ def handle_non_complete_credit_card_refunds(cls):
                 elif cls._is_status_paid_and_invoice_refund_requested(status, invoice):
                     cls._refund_paid(invoice)
                 elif cls._is_status_complete(status):
-                    if status.refundstatus is None:
```

Comment:
Manual refunds don't exist anymore, this was needed for when we first initially launched.